File: .snippets/code/chain-interactions/accounts/create-account/create-account.ts
import { cryptoWaitReady, mnemonicGenerate } from '@polkadot/util-crypto';
import { Keyring } from '@polkadot/keyring';

async function main() {
  await cryptoWaitReady();

  const mnemonic = mnemonicGenerate(12);
  const keyring = new Keyring({ type: 'sr25519', ss58Format: 0 });
  const pair = keyring.addFromMnemonic(mnemonic);

  console.log(`Address: ${pair.address}`);
  console.log(`Mnemonic: ${mnemonic}`);
}

main().catch(console.error);

File: docs/.snippets/code/chain-interactions/accounts/create-account/create-account.rs
use sp_core::{crypto::Ss58Codec, Pair};

fn main() {
    let (pair, phrase, _) = sp_core::sr25519::Pair::generate_with_phrase(None);
    let address = pair.public().to_ss58check();
    
    println!("Address: {}", address);
    println!("Mnemonic: {}", phrase);
}